src/core/shards/core.hpp

```diff
@@ -1192,9 +1192,15 @@ struct Swap {
     _targetB = referenceVariable(ctx, _nameB.c_str());
   }
 
+  static SHOptionalString help() {
+    return SHCCSTR("Swaps the values of the two variables passed to it via `:NameA` and `:NameB` parameters.");
+  }
+
   static SHTypesInfo inputTypes() { return CoreInfo::AnyType; }
+  static SHOptionalString inputHelp() { return SHCCSTR("NA"); }
 
   static SHTypesInfo outputTypes() { return CoreInfo::AnyType; }
+  static SHOptionalString outputHelp() { return SHCCSTR("NA"); }
```

Comment:
Since it is used in several places, I wonder whether it should it be declared as a const. @sinkingsugar can `SHOptionalString` be `constexpr`?

Or maybe it doesn't matter since it will be compiled into the same string given its crc32.

Reply:
Shall we resolve this conversation, then?